src/modules/client/components/config/ServiceEditionModal.vue

```diff
@@ -16,11 +16,17 @@
       type="number" @blur="validations.vat.$touch" error-message="La TVA doit être positive ou nulle"
       @input="update($event, 'vat')" />
     <ni-select in-modal v-if="editedService.nature !== FIXED" caption="Plan de majoration"
-      :value="editedService.surcharge" :options="surchargesOptions" clearable @input="update($event, 'surcharge')" />
+      :value="editedService.surcharge" :options="surchargesOptions" @input="update($event, 'surcharge')" />
     <div class="row q-mb-md">
       <q-checkbox label="Exonération de charges" :value="editedService.exemptFromCharges" dense
         @input="update($event, 'exemptFromCharges')" />
     </div>
+    <div class="row" v-for="(billingItem, index) in editedService.billingItems" :key="index">
+      <ni-select in-modal :full-width="false" :reset-button="false" :value="billingItem" :options="billingItemsOptions"
```

Comment:
je ne comprends pas a quoi serve les options `full-width` et `reset-button`

Reply:
plutot que resetButton, quasar a une props qui s'appelle `clearable` que tu pourrais utiliser je pense

Reply:
full-width fait en sorte que le composant prennent 100% de la largeur de la modal (c'est pourquoi on l'avait mis à true sur inModal). Ici, j'ai besoin d'ajouter un bouton à côté du Select et le tout dans une modal. Je veux donc décorréler inModal et full-width

Reply:
la props clearable n'a pas l'air de fonctionner, je pense que ça vient du fait qu'on remplace le bouton de clear dans notre composant NiSelect

Reply:
si tu vaildes ce que j'ai fait dans ma PR, tu pourras t'en inspirer

Reply:
suite a mes changement, tu peux renommer in-modal en full-width en fait non ? car in-modal n'est utilisé que pour ca non ?
(tu peux remplacer `:class="{'borders-parent': inModal}"` par `class="borders-parent"`car le `type="file"` n'est utilisé que dans des modales

Reply:
et du coup tu n'as plus besoin de full-width je pense

Reply:
done